src/applications/vaos/tests/new-appointment/components/ConfirmationPage/index.unit.spec.jsx

```diff
@@ -1,138 +1,158 @@
-import React from 'react';
-import { shallow } from 'enzyme';
+import userEvent from '@testing-library/user-event';
```

Comment:
One thing we might consider is consolidating these tests into just this file, rather than three separate ones, and always render the `ConfirmationPage` component when testing. I want to get away from unit testing all the components individually and instead test them at the page level, so we make sure they work correctly together.

Reply:
Won't the e2e tests cover this? I think we have to figure out where the line is between unit and e2e test. I wonder what happens with code coverage with this approach.

Reply:
While it's not a perfectly clear line, I think:

- Cypress tests should test that a user can progress through the major paths through the form and successfully submit at the end
- React Testing Library tests should test specific behavior and presentation within each page (and certain interactions between pages that are hard to test in Cypress).

Cypress tests don't currently factor into our code coverage metrics, but anything tested in our "unit" tests will. RTL will render a component and any sub-components, so all code will get exercised.

Reply:
Done.